### DISCUSSION TITLE
faster-whisper

### DISCUSSION THREAD

**Discussion by jack-tol** [flet-dev/flet#3658] [Packages] [answered]
This would be an amazing integration for the mobile app i'm developing!

**Comment by ndonkoHenri** [comment, marked as answer]
`faster-whisper` can't be packaged for Flet mobile because its entire native stack is unbuildable for iOS/Android:

- `ctranslate2` (the inference engine) — no source distribution on PyPI and no existing mobile build.
- `onnxruntime` (used for the Silero VAD) — no source distribution; only desktop/server wheels.
- av/PyAV (audio decoding) — needs a full FFmpeg native chain that isn't available here.
